packages/web-runtime/src/components/Topbar/UserMenu.vue

```diff
@@ -70,17 +71,21 @@
           </div>
         </li>
       </oc-list>
+      <div v-if="imprintUrl && privacyUrl" class="imprint-footer oc-py-s oc-mt-m oc-text-center">
```

Comment:
```suggestion
      <div v-if="imprintUrl || privacyUrl" class="imprint-footer oc-py-s oc-mt-m oc-text-center">
```

... and a `v-if` with the respective url on the two oc-buttons please. If any of the links exists we want to show it. If someone for whatever reason only has an imprint, but not a privacy url, we still want to show the imprint link.

Reply:
ah okay makes sense, will build it so only one can be shown